### PR TITLE
feat: add attributes to stake page tracks

### PR DESCRIPTION
**Pull request by sarneijim**
### 📝 Description

Add attribute to stake page tracks 

### ❓ Context

- **Impacted projects**: LLD
- **Linked resource(s)**: https://ledgerhq.atlassian.net/browse/LIVE-7837

### ✅ Checklist

- [ ] **Test coverage** 
- [ ] **Atomic delivery** 
- [ ] **No breaking changes** 

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

### CODE REVIEW

File: apps/ledger-live-desktop/src/renderer/families/tezos/DelegateFlowModal/steps/StepAccount.tsx [hunk 1]
```diff
@@ -29,6 +29,9 @@ const StepAccount = ({
       <TrackPage
         category={`Delegation Flow${eventType ? ` (${eventType})` : ""}`}
         name="Step Account"
+        flow="stake"
+        action="delegation"
+        currency="tezos"
```
Comment (kallen-ledger) [inline apps/ledger-live-desktop/src/renderer/families/tezos/DelegateFlowModal/steps/StepAccount.tsx:34]: tezos or xtz?

Comment (sarneijim) [reply apps/ledger-live-desktop/src/renderer/families/tezos/DelegateFlowModal/steps/StepAccount.tsx:34]: Great eye!

**Review by kallen-ledger — APPROVED** [review]

File: apps/ledger-live-desktop/src/renderer/families/celo/ActivateFlowModal/steps/StepVote.tsx [hunk 2]
```diff
@@ -84,7 +84,13 @@ const StepVote = ({
   const unit = getAccountUnit(account);
   return (
     <Box flow={1}>
-      <TrackPage category="Celo Activate" name="Step 1" />
+      <TrackPage
+        category="Celo Activate"
+        name="Step 1"
```
Comment (mitchellv-ledger) [inline apps/ledger-live-desktop/src/renderer/families/celo/ActivateFlowModal/steps/StepVote.tsx:89]: would it not be better for this to be `Step Vote?

File: apps/ledger-live-desktop/src/renderer/families/celo/SimpleOperationFlowModal/steps/StepInfo.tsx [hunk 3]
```diff
@@ -45,7 +45,13 @@ const StepInfo = ({ account, transaction, warning, error, t, mode }: StepProps)
   const description = t(`celo.simpleOperation.modes.${mode}.description`);
   return (
     <Box flow={1}>
-      <TrackPage category="Celo SimpleOperation" name="Step 1" />
+      <TrackPage
+        category="Celo SimpleOperation"
+        name="Step 1"
```
Comment (mitchellv-ledger) [inline apps/ledger-live-desktop/src/renderer/families/celo/SimpleOperationFlowModal/steps/StepInfo.tsx:50]: Should this be `Step Info`

File: apps/ledger-live-desktop/src/renderer/families/celo/UnlockFlowModal/steps/StepAmount.tsx [hunk 4]
```diff
@@ -56,7 +56,13 @@ const StepAmount = ({
   return (
     <Box flow={1}>
       <SyncSkipUnderPriority priority={100} />
-      <TrackPage category="Celo Unlock" name="Step 1" />
+      <TrackPage
+        category="Celo Unlock"
+        name="Step 1"
```
Comment (mitchellv-ledger) [inline apps/ledger-live-desktop/src/renderer/families/celo/UnlockFlowModal/steps/StepAmount.tsx:61]: `Step Amount`?

File: apps/ledger-live-desktop/src/renderer/families/celo/WithdrawFlowModal/steps/StepAmount.tsx [hunk 5]
```diff
@@ -81,7 +81,13 @@ const StepAmount = ({
     onChange(pendingWithdrawals[0].index);
   return (
     <Box flow={1}>
-      <TrackPage category="Celo Withdraw" name="Step 1" />
+      <TrackPage
+        category="Celo Withdraw"
+        name="Step 1"
```
Comment (mitchellv-ledger) [inline apps/ledger-live-desktop/src/renderer/families/celo/WithdrawFlowModal/steps/StepAmount.tsx:86]: `Step Amount`?

File: apps/ledger-live-desktop/src/renderer/families/cosmos/ClaimRewardsFlowModal/steps/StepClaimRewards.tsx [hunk 6]
```diff
@@ -69,7 +69,13 @@ export default function StepClaimRewards({
   const key = transaction.mode === "claimReward" ? "claimInfo" : "compoundInfo";
   return (
     <Box flow={1}>
-      <TrackPage category="ClaimRewards Flow" name="Step 1" />
+      <TrackPage
+        category="ClaimRewards Flow"
+        name="Step 1"
```
Comment (mitchellv-ledger) [inline apps/ledger-live-desktop/src/renderer/families/cosmos/ClaimRewardsFlowModal/steps/StepClaimRewards.tsx:74]: `Step Claim Rewards`?

File: apps/ledger-live-desktop/src/renderer/families/cosmos/RedelegationFlowModal/steps/StepValidators.tsx [hunk 7]
```diff
@@ -142,7 +142,13 @@ export default function StepValidators({
   const crypto = cryptoFactory(account.currency.id);
   return (
     <Container>
-      <TrackPage category="Redelegation Flow" name="Step 1" />
+      <TrackPage
+        category="Redelegation Flow"
+        name="Step 1"
```
Comment (mitchellv-ledger) [inline apps/ledger-live-desktop/src/renderer/families/cosmos/RedelegationFlowModal/steps/StepValidators.tsx:147]: `Step Validators`?

File: apps/ledger-live-desktop/src/renderer/families/cosmos/UndelegationFlowModal/steps/Amount.tsx [hunk 8]
```diff
@@ -70,7 +70,13 @@ export default function StepAmount({
   const crypto = cryptoFactory(account.currency.id);
   return (
     <Box flow={1}>
-      <TrackPage category="Undelegation Flow" name="Step 1" />
+      <TrackPage
+        category="Undelegation Flow"
+        name="Step 1"
```
Comment (mitchellv-ledger) [inline apps/ledger-live-desktop/src/renderer/families/cosmos/UndelegationFlowModal/steps/Amount.tsx:75]: `Step Amount`?

File: apps/ledger-live-desktop/src/renderer/families/elrond/components/Modals/Claim/steps/StepClaimRewards.tsx [hunk 9]
```diff
@@ -58,7 +58,13 @@ const StepClaimRewards = (props: StepProps) => {
   const key = transaction.mode === "claimRewards" ? "claimInfo" : "compoundInfo";
   return (
     <Box flow={1}>
-      <TrackPage category="ClaimRewards Flow" name="Step 1" />
+      <TrackPage
+        category="ClaimRewards Flow"
+        name="Step 1"
```
Comment (mitchellv-ledger) [inline apps/ledger-live-desktop/src/renderer/families/elrond/components/Modals/Claim/steps/StepClaimRewards.tsx:63]: `Step Claim Rewards`?

File: apps/ledger-live-desktop/src/renderer/families/elrond/components/Modals/Undelegate/steps/Amount.tsx [hunk 10]
```diff
@@ -60,7 +60,13 @@ const StepAmount = (props: StepProps) => {
   if (!account) return null;
   return (
     <Box flow={1}>
-      <TrackPage category="Undelegation Flow" name="Step 1" />
+      <TrackPage
+        category="Undelegation Flow"
+        name="Step 1"
```
Comment (mitchellv-ledger) [inline apps/ledger-live-desktop/src/renderer/families/elrond/components/Modals/Undelegate/steps/Amount.tsx:65]: `Step Amount`?

File: apps/ledger-live-desktop/src/renderer/families/elrond/components/Modals/Withdraw/steps/StepWithdraw.tsx [hunk 11]
```diff
@@ -43,7 +43,13 @@ const StepWithdraw = (props: StepProps) => {
   );
   return (
     <Box flow={1}>
-      <TrackPage category="ClaimRewards Flow" name="Step 1" />
+      <TrackPage
+        category="ClaimRewards Flow"
+        name="Step 1"
```
Comment (mitchellv-ledger) [inline apps/ledger-live-desktop/src/renderer/families/elrond/components/Modals/Withdraw/steps/StepWithdraw.tsx:48]: `Step Withdraw`?

File: apps/ledger-live-desktop/src/renderer/families/near/UnstakingFlowModal/steps/Amount.tsx [hunk 12]
```diff
@@ -65,7 +65,13 @@ export default function StepAmount({
   const amount = useMemo(() => (validator ? validator.amount : new BigNumber(0)), [validator]);
   return (
     <Box flow={1}>
-      <TrackPage category="Unstaking Flow" name="Step 1" />
+      <TrackPage
+        category="Unstaking Flow"
+        name="Step 1"
```
Comment (mitchellv-ledger) [inline apps/ledger-live-desktop/src/renderer/families/near/UnstakingFlowModal/steps/Amount.tsx:70]: `Step Amount`?

File: apps/ledger-live-desktop/src/renderer/families/near/WithdrawingFlowModal/steps/Amount.tsx [hunk 13]
```diff
@@ -62,7 +62,13 @@ export default function StepAmount({
   const amount = useMemo(() => (validator ? validator.amount : new BigNumber(0)), [validator]);
   return (
     <Box flow={1}>
-      <TrackPage category="Withdrawing Flow" name="Step 1" />
+      <TrackPage
+        category="Withdrawing Flow"
+        name="Step 1"
```
Comment (mitchellv-ledger) [inline apps/ledger-live-desktop/src/renderer/families/near/WithdrawingFlowModal/steps/Amount.tsx:67]: `Step Amount`?

File: apps/ledger-live-desktop/src/renderer/families/polkadot/BondFlowModal/steps/StepAmount.tsx [hunk 14]
```diff
@@ -41,7 +41,7 @@ export default function StepAmount({
   return (
     <Box flow={1}>
       <SyncSkipUnderPriority priority={100} />
-      <TrackPage category="Bond Flow" name="Step 1" />
+      <TrackPage category="Bond Flow" name="Step 1" flow="stake" action="bond" currency="dot" />
```
Comment (mitchellv-ledger) [inline apps/ledger-live-desktop/src/renderer/families/polkadot/BondFlowModal/steps/StepAmount.tsx:44]: `Step Amount`?

File: apps/ledger-live-desktop/src/renderer/families/polkadot/NominationFlowModal/steps/StepNomination.tsx [hunk 15]
```diff
@@ -42,7 +42,13 @@ export default function StepNomination({
   const nominations = polkadotResources.nominations || [];
   return (
     <Box flow={1}>
-      <TrackPage category="Nomination Flow" name="Step 1" />
+      <TrackPage
+        category="Nomination Flow"
+        name="Step 1"
```
Comment (mitchellv-ledger) [inline apps/ledger-live-desktop/src/renderer/families/polkadot/NominationFlowModal/steps/StepNomination.tsx:47]: `Step Nomination`

File: apps/ledger-live-desktop/src/renderer/families/polkadot/RebondFlowModal/steps/StepAmount.tsx [hunk 16]
```diff
@@ -24,7 +24,7 @@ export default function StepAmount({
   return (
     <Box flow={1}>
       <SyncSkipUnderPriority priority={100} />
-      <TrackPage category="Bond Flow" name="Step 1" />
+      <TrackPage category="Bond Flow" name="Step 1" flow="stake" action="rebond" currency="dot" />
```
Comment (mitchellv-ledger) [inline apps/ledger-live-desktop/src/renderer/families/polkadot/RebondFlowModal/steps/StepAmount.tsx:27]: `Step Amount`

File: apps/ledger-live-desktop/src/renderer/families/polkadot/SimpleOperationFlowModal/steps/StepInfo.tsx [hunk 17]
```diff
@@ -15,9 +15,16 @@ export default function StepInfo({ account, transaction, warning, error, mode }:
     account && account.polkadotResources && transaction,
     "account and transaction required",
   );
+  const action = mode.replace(/([A-Z])/g, "_$1").toLowerCase();
   return (
     <Box flow={1}>
-      <TrackPage category="SimpleOperationFlow Flow" name="Step 1" />
+      <TrackPage
+        category="SimpleOperationFlow Flow"
+        name="Step 1"
```
Comment (mitchellv-ledger) [inline apps/ledger-live-desktop/src/renderer/families/polkadot/SimpleOperationFlowModal/steps/StepInfo.tsx:23]: `Step Info`?

File: apps/ledger-live-desktop/src/renderer/families/polkadot/UnbondFlowModal/steps/StepAmount.tsx [hunk 18]
```diff
@@ -24,7 +24,13 @@ export default function StepAmount({
   return (
     <Box flow={1}>
       <SyncSkipUnderPriority priority={100} />
-      <TrackPage category="Unbond Flow" name="Step 1" />
+      <TrackPage
+        category="Unbond Flow"
+        name="Step 1"
```
Comment (mitchellv-ledger) [inline apps/ledger-live-desktop/src/renderer/families/polkadot/UnbondFlowModal/steps/StepAmount.tsx:29]: `Step Amount`

**Review by mitchellv-ledger — COMMENTED** [review]
I would assume you'd probably want something more descriptive than `Step 1` however if that's what you want feel free to ignore all comments

File: apps/ledger-live-desktop/src/renderer/families/celo/ActivateFlowModal/steps/StepVote.tsx (hunk 2 above)
Comment (sarneijim) [reply apps/ledger-live-desktop/src/renderer/families/celo/ActivateFlowModal/steps/StepVote.tsx:89]: These are not our attributes. I understand blockchain has followed analytics specifications.

**Pull request merged**